models/marts/int_custoemrs_no_orders.sql

```diff
@@ -0,0 +1,22 @@
+{%- set status = ['return_pending','returned','complete'] -%}
+with customers as (
+   select * from {{ ref('stg_customers') }}
+),
+orders as (
+   select * from {{ ref('stg_orders') }}
+),
+ 
+final as (
+   select c.customer_id,
+     {% for status in status -%}

```

Comment:
I recommend having separate words for your loops so you have a better sense of what is looping through what. So in this case, I would likely use `{% for status in statuses -%}` or something like that. And have this `{%- set status = ['return_pending','returned','complete'] -%}` be `{%- set statuses = ['return_pending','returned','complete'] -%}`